setup.cfg

```diff
@@ -11,5 +11,8 @@ ignore =
     # mutual exclusive with W504
     W503
 
-[mypy-influxdb.*,elasticsearch.*,fqn_decorators.*,pkgsettings.*,kafka.*,setuptools.*,Queue.*]
+[mypy]
+ignore_missing_imports = true
+
+[mypy-mock,freezegun]
```

Comment:
FYI, `mypy` now supports `pyprohect.toml`

and it looks better when placed there:
```
[tool.mypy]
ignore_missing_imports = true

[[tool.mypy.overrides]]
module = [
  "mock",
  "freezegun",
]
ignore_missing_imports = true
```

BTW, why do we need to enable `ignore_missing_imports` for both global mypy configuration and these two modules? shouldn't it be enough to have only global one?

Reply:
Good one. Looks better now I think.